modules/gitcoinPassportEligibility.json

```diff
@@ -0,0 +1,252 @@
+{
+  "name": "Gitcoin Passport Eligibility",
+  "details": [
+    "A Hats Protocol eligibility module that checks if an address has a minimum score that passes the GitcoinPassportDecoder or custom threshold."
+  ],
+  "links": [
+    {
+      "label": "GitHub",
+      "link": "https://github.com/daocoa/gitcoin-passport-eligibility/"
+    }
+  ],
+  "parameters": [
+    {
+      "label": "Gitcoin Passport Decoder",
+      "functionName": "gitcoinPassportDecoder",
+      "displayType": "default",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as an address in the module smart contract. Is this correct or should I use `token`?"
+
+    },
+    {
+      "label": "Score Criterion",
+      "functionName": "scoreCriterion",
+      "displayType": "default",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as a uint256 in the module smart contract. Is this correct or should I use `amountWithDecimals` or `hat`?"
+    }
+  ],
+  "type": {
+    "eligibility": true,
+    "toggle": false,
+    "hatter": false
+  },
+  "tags": [],
+  "implementationAddress": "0x4Ce2132f7cC5d5625D757CeDBE67D26CB4C0ec0f",
+  "deployments": [
+    {
+      "chainId": "10",
+      "block": "TBA"
+    },
+    {
+      "chainId": "11155420",
+      "block": "11734099"
+    },
+    {
+      "chainId": "11155111",
+      "block": "5868541"
+    }
+  ],
+  "creationArgs": {
+    "useHatId": true,
+    "immutable": [
+      {
+        "name": "Gitcoin Passport Decoder",
+        "description": "Decoder address",
+        "type": "address",
+        "example": "0xe53C60F8069C2f0c3a84F9B3DB5cf56f3100ba56",
+        "displayType": "address",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as an address in the module smart contract. Is this correct or should I use `token`?"
+
+      },
+      {
+        "name": "Score Criterion",
+        "description": "Minimum threshold of passport to be considered eligible.",
+        "type": "uint256",
+        "example": "10",
+        "displayType": "amount",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as a uint256 in the module smart contract. Is this correct or should I use `amount`, `amountWithDecimals` or `hat`? I do not see `amount` in the modules-registry README anywhere."
```

Comment:
The `amountWithDecimals` display type is used for displaying token balances and taking into consideration their `decimals` value. The decoder is not implementing a token standard and so the  `amountWithDecimals` type will not work in this case. @spengrah is it right that the decoder's resolution is 4 decimals? If so, then @scottrepreneur do you think we'll want a dedicated display type for this?

Reply:
After chatting with Ido, we're suggesting we'll extend the generic `amountWithDecimals` displayType to include an optional `decimals` parameter. Coming back with an example for this implementation here.

Reply:
Realized now that using an object will cause the schema validation test to fail, because it's expecting a `string` type for the `displayType` property. We can either:
- ignore this test in this pr and then create a pr that adjusts the schema to allow an object for the `displayType` prop
- suggest including the schema adjustment in this pr 
- suggest a temporal value like `amount-4` with the convention that 4 means decimals and modify it later

What do you think? I think i'll pick option (1) here

Reply:
1 seems great

Reply:
Option 1 would be a great solution!

Reply:
So for the display type of the "Score Criterion" property, I think we can suggest using:
```json
{
  "type": "amount",
  "decimals": 4
}
```
Looks good @scottrepreneur ?

Reply:
the type should still be `amountWithDecimals` imo. otherwise looks good

Reply:
Great! I have updated the PR to accommodate for this design.